app/models/miq_server/status_management.rb

```diff
@@ -3,14 +3,39 @@
 module MiqServer::StatusManagement
   extend ActiveSupport::Concern
 
+  def status_update
+    self.attributes = system_status
+    self.attributes = process_status
+    save!
```

Comment:
Yeah I was confused about what `attributes=` does. As @jrafanie just pointed out to me, it's alias, `.assign_attributes()`, is at least a bit more clear.